Microsoft.NET.Build.Containers/ContainerBuilder.cs

```diff
@@ -25,8 +29,8 @@ public static async Task Containerize(DirectoryInfo folder, string workingDir, s
 
         img.SetEntrypoint(entrypoint, entrypointArgs);
 
-        var isDockerPush = outputRegistry.StartsWith("docker://");
-        Registry? outputReg = isDockerPush ? null : new Registry(new Uri(outputRegistry));
+        var isDockerPush = String.IsNullOrEmpty(outputRegistry);
```

Comment:
Should this also consider the user manually setting `docker.io` rather than just checking empty?

Reply:
Setting docker.io would mean `push to docker hub` now in this world.